### DISCUSSION TITLE
spacy installation failed

### DISCUSSION THREAD

**Discussion by alexibexi31** [explosion/spaCy#9099] [Help: Installation] [answered]
Hey,
I'm a windows user and tried to install spacy via pip in my terminal. I did it with "python -m pip install spacy" and then downloaded my package "python -m spacy download de_core_news_sm". After that  I did "pip list" and it is installed, but if i run my programme there appears an error "no module named spacy".
I don't know what to do.
During installation there was a warning that says: WARNING: The script spacy.exe is installed in ‘C:\Users\Alexa\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\Scripts’ which is not on PATH.
Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Maybe there is the mistake?
I would appreciate any help
Thank you!

**Comment by svlandeg** [comment, marked as answer]
Hi!

Did you start the installation in a new, clean virtual environment?
